docs/command_line_reference.rst

```diff
@@ -41,68 +44,99 @@ command above).
-    ``<instance>`` is the instance name as displayed on the command-line.
-:``--experiment <e>``: selects the experiment named ``e``.
-:``--failed``: selects all the failed experiments.
-:``--instance <i>``: selects all experiments with the instance named ``i``.
-:``--instset <i>``: selects all experiments with the instance set named ``i``.
-:``--unfinished``: selects all the unfinished experiments.
-:``--revision <i>``: selects all experiments with the revision named ``r``.
-:``--variants [variants...]``: selects all experiments with the variants from the space separated list of variants.
+:--all:                    Selects all the experiments.
+:--axes [axes...]:         Selects all experiments with the variant axes from
+                           the space separated list of *axes*.
+:--run <r>:                Selects the single run given as
+                           ``<experiment_display_name>/<instance>``, where
+                           ``<experiment_display_name>`` is the name of the
+                           experiment as displayed on the command line and
+                           ``<instance>`` is the instance name as displayed on
+                           the command line.
+:--experiment <e>:         Selects the experiment named *e*.
+:--failed:                 Selects all the failed experiments.
+:--instance <i>:           Selects all experiments with the instance named *i*.
+:--instset <i>:            Selects all experiments with the instance set named *i*.
+:--unfinished:             Selects all the unfinished experiments.
+:--revision <i>:           Selects all experiments with the revision named *r*.
+:--variants [variants...]: Selects all experiments with the variants from the
+                           space separated list of variants.
 
 instances
 ---------
-Checks and eventually downloads the instances for the experiments.
-It supports the following actions:
 
-:install: downloads all the missing instances if they are taken from a public repository.
-   With the argument ``--overwrite`` it will also download the available instances and
-   overwrite them.
-:list: lists all the instances found in the experiments.yml file.
-   Available instances are shown in green, unavailable instances in red.
-:process: caches information about instances.
-:run-transform: manually runs a transformation on instance files.
+The instance command can check the availability, and install instances. It
+supports the following flags:
+
+:install:         Installs missing instances. This can also trigger a download if an
+                  instance resource is configured accordingly. With the argument
+                  ``--overwrite``, this command will reinstall all instances and
+                  overwrite existing ones.
+
+:list:            Lists all defined instances. Available instances are shown in green,
+                  unavailable instances in red.
+
+:process:         Caches information about instances.
```

Comment:
very unclear explanation. what is cached here?

Reply:
Please look at new new change. I will create a new issue to actually remove this functionality. We do not only use NetworKit for this feature, which brings in another dependency, but is also very specific for graph files. I don't think that the value of this feature justifies the NetworKit dependency and maintenance of the feature.

Reply:
I like the new explanation and I agree that this functionality is too specific.